29.10.2008 17:26:06 *ERROR* [main] HTTPConnection: Send request error (HTTPConnection.java, line 2933)
java.net.ConnectException: Connection refused
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.PlainSocketImpl.doConnect(PlainSocketImpl.java:333)
	at java.net.PlainSocketImpl.connectToAddress(PlainSocketImpl.java:195)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:182)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:366)
	at java.net.Socket.connect(Socket.java:520)
	at java.net.Socket.connect(Socket.java:470)
	at java.net.Socket.<init>(Socket.java:367)
	at java.net.Socket.<init>(Socket.java:209)
	at org.exoplatform.common.http.client.HTTPConnection.getSocket(HTTPConnection.java:3046)
	at org.exoplatform.common.http.client.HTTPConnection.sendRequest(HTTPConnection.java:2841)
	at org.exoplatform.common.http.client.HTTPConnection.handleRequest(HTTPConnection.java:2700)
	at org.exoplatform.common.http.client.HTTPConnection.setupRequest(HTTPConnection.java:2527)
	at org.exoplatform.common.http.client.HTTPConnection.Put(HTTPConnection.java:1063)
	at org.exoplatform.common.http.client.HTTPConnection.Put(HTTPConnection.java:1033)
	at org.exoplatform.common.http.client.HTTPConnection.Put(HTTPConnection.java:1012)
	at org.exoplatform.services.jcr.webdav.command.TestCopy.setUp(TestCopy.java:60)
	at junit.framework.TestCase.runBare(TestCase.java:125)
	at junit.framework.TestResult$1.protect(TestResult.java:106)
	at junit.framework.TestResult.runProtected(TestResult.java:124)
	at junit.framework.TestResult.run(TestResult.java:109)
	at junit.framework.TestCase.run(TestCase.java:118)
	at junit.framework.TestSuite.runTest(TestSuite.java:208)
	at junit.framework.TestSuite.run(TestSuite.java:203)
	at org.eclipse.jdt.internal.junit.runner.junit3.JUnit3TestReference.run(JUnit3TestReference.java:130)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:460)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:673)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:386)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:196)
29.10.2008 17:26:06 *ERROR* [main] HTTPConnection: Send request error (HTTPConnection.java, line 2933)
java.net.ConnectException: Connection refused
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.PlainSocketImpl.doConnect(PlainSocketImpl.java:333)
	at java.net.PlainSocketImpl.connectToAddress(PlainSocketImpl.java:195)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:182)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:366)
	at java.net.Socket.connect(Socket.java:520)
	at java.net.Socket.connect(Socket.java:470)
	at java.net.Socket.<init>(Socket.java:367)
	at java.net.Socket.<init>(Socket.java:209)
	at org.exoplatform.common.http.client.HTTPConnection.getSocket(HTTPConnection.java:3046)
	at org.exoplatform.common.http.client.HTTPConnection.sendRequest(HTTPConnection.java:2841)
	at org.exoplatform.common.http.client.HTTPConnection.handleRequest(HTTPConnection.java:2700)
	at org.exoplatform.common.http.client.HTTPConnection.setupRequest(HTTPConnection.java:2527)
	at org.exoplatform.common.http.client.HTTPConnection.Put(HTTPConnection.java:1063)
	at org.exoplatform.common.http.client.HTTPConnection.Put(HTTPConnection.java:1033)
	at org.exoplatform.common.http.client.HTTPConnection.Put(HTTPConnection.java:1012)
	at org.exoplatform.services.jcr.webdav.command.TestCopy.setUp(TestCopy.java:60)
	at junit.framework.TestCase.runBare(TestCase.java:125)
	at junit.framework.TestResult$1.protect(TestResult.java:106)
	at junit.framework.TestResult.runProtected(TestResult.java:124)
	at junit.framework.TestResult.run(TestResult.java:109)
	at junit.framework.TestCase.run(TestCase.java:118)
	at junit.framework.TestSuite.runTest(TestSuite.java:208)
	at junit.framework.TestSuite.run(TestSuite.java:203)
	at org.eclipse.jdt.internal.junit.runner.junit3.JUnit3TestReference.run(JUnit3TestReference.java:130)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:460)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:673)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:386)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:196)
29.10.2008 17:26:06 *ERROR* [main] HTTPConnection: Send request error (HTTPConnection.java, line 2933)
java.net.ConnectException: Connection refused
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.PlainSocketImpl.doConnect(PlainSocketImpl.java:333)
	at java.net.PlainSocketImpl.connectToAddress(PlainSocketImpl.java:195)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:182)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:366)
	at java.net.Socket.connect(Socket.java:520)
	at java.net.Socket.connect(Socket.java:470)
	at java.net.Socket.<init>(Socket.java:367)
	at java.net.Socket.<init>(Socket.java:209)
	at org.exoplatform.common.http.client.HTTPConnection.getSocket(HTTPConnection.java:3046)
	at org.exoplatform.common.http.client.HTTPConnection.sendRequest(HTTPConnection.java:2841)
	at org.exoplatform.common.http.client.HTTPConnection.handleRequest(HTTPConnection.java:2700)
	at org.exoplatform.common.http.client.HTTPConnection.setupRequest(HTTPConnection.java:2527)
	at org.exoplatform.common.http.client.HTTPConnection.Put(HTTPConnection.java:1063)
	at org.exoplatform.common.http.client.HTTPConnection.Put(HTTPConnection.java:1033)
	at org.exoplatform.common.http.client.HTTPConnection.Put(HTTPConnection.java:1012)
	at org.exoplatform.services.jcr.webdav.command.TestCopy.setUp(TestCopy.java:60)
	at junit.framework.TestCase.runBare(TestCase.java:125)
	at junit.framework.TestResult$1.protect(TestResult.java:106)
	at junit.framework.TestResult.runProtected(TestResult.java:124)
	at junit.framework.TestResult.run(TestResult.java:109)
	at junit.framework.TestCase.run(TestCase.java:118)
	at junit.framework.TestSuite.runTest(TestSuite.java:208)
	at junit.framework.TestSuite.run(TestSuite.java:203)
	at org.eclipse.jdt.internal.junit.runner.junit3.JUnit3TestReference.run(JUnit3TestReference.java:130)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:460)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:673)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:386)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:196)
29.10.2008 17:26:06 *ERROR* [main] HTTPConnection: Send request error (HTTPConnection.java, line 2933)
java.net.ConnectException: Connection refused
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.PlainSocketImpl.doConnect(PlainSocketImpl.java:333)
	at java.net.PlainSocketImpl.connectToAddress(PlainSocketImpl.java:195)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:182)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:366)
	at java.net.Socket.connect(Socket.java:520)
	at java.net.Socket.connect(Socket.java:470)
	at java.net.Socket.<init>(Socket.java:367)
	at java.net.Socket.<init>(Socket.java:209)
	at org.exoplatform.common.http.client.HTTPConnection.getSocket(HTTPConnection.java:3046)
	at org.exoplatform.common.http.client.HTTPConnection.sendRequest(HTTPConnection.java:2841)
	at org.exoplatform.common.http.client.HTTPConnection.handleRequest(HTTPConnection.java:2700)
	at org.exoplatform.common.http.client.HTTPConnection.setupRequest(HTTPConnection.java:2527)
	at org.exoplatform.common.http.client.HTTPConnection.Put(HTTPConnection.java:1063)
	at org.exoplatform.common.http.client.HTTPConnection.Put(HTTPConnection.java:1033)
	at org.exoplatform.common.http.client.HTTPConnection.Put(HTTPConnection.java:1012)
	at org.exoplatform.services.jcr.webdav.command.TestCopy.setUp(TestCopy.java:60)
	at junit.framework.TestCase.runBare(TestCase.java:125)
	at junit.framework.TestResult$1.protect(TestResult.java:106)
	at junit.framework.TestResult.runProtected(TestResult.java:124)
	at junit.framework.TestResult.run(TestResult.java:109)
	at junit.framework.TestCase.run(TestCase.java:118)
	at junit.framework.TestSuite.runTest(TestSuite.java:208)
	at junit.framework.TestSuite.run(TestSuite.java:203)
	at org.eclipse.jdt.internal.junit.runner.junit3.JUnit3TestReference.run(JUnit3TestReference.java:130)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:460)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:673)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:386)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:196)
29.10.2008 17:54:22 *ERROR* [main] HTTPConnection: Send request error (HTTPConnection.java, line 2933)
java.net.ConnectException: Connection refused
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.PlainSocketImpl.doConnect(PlainSocketImpl.java:333)
	at java.net.PlainSocketImpl.connectToAddress(PlainSocketImpl.java:195)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:182)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:366)
	at java.net.Socket.connect(Socket.java:520)
	at java.net.Socket.connect(Socket.java:470)
	at java.net.Socket.<init>(Socket.java:367)
	at java.net.Socket.<init>(Socket.java:209)
	at org.exoplatform.common.http.client.HTTPConnection.getSocket(HTTPConnection.java:3046)
	at org.exoplatform.common.http.client.HTTPConnection.sendRequest(HTTPConnection.java:2841)
	at org.exoplatform.common.http.client.HTTPConnection.handleRequest(HTTPConnection.java:2700)
	at org.exoplatform.common.http.client.HTTPConnection.setupRequest(HTTPConnection.java:2527)
	at org.exoplatform.common.http.client.HTTPConnection.Put(HTTPConnection.java:1063)
	at org.exoplatform.common.http.client.HTTPConnection.Put(HTTPConnection.java:1033)
	at org.exoplatform.common.http.client.HTTPConnection.Put(HTTPConnection.java:1012)
	at org.exoplatform.services.jcr.webdav.command.TestCopy.setUp(TestCopy.java:60)
	at junit.framework.TestCase.runBare(TestCase.java:125)
	at junit.framework.TestResult$1.protect(TestResult.java:106)
	at junit.framework.TestResult.runProtected(TestResult.java:124)
	at junit.framework.TestResult.run(TestResult.java:109)
	at junit.framework.TestCase.run(TestCase.java:118)
	at junit.framework.TestSuite.runTest(TestSuite.java:208)
	at junit.framework.TestSuite.run(TestSuite.java:203)
	at org.eclipse.jdt.internal.junit.runner.junit3.JUnit3TestReference.run(JUnit3TestReference.java:130)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:460)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:673)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:386)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:196)
29.10.2008 17:54:22 *ERROR* [main] HTTPConnection: Send request error (HTTPConnection.java, line 2933)
java.net.ConnectException: Connection refused
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.PlainSocketImpl.doConnect(PlainSocketImpl.java:333)
	at java.net.PlainSocketImpl.connectToAddress(PlainSocketImpl.java:195)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:182)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:366)
	at java.net.Socket.connect(Socket.java:520)
	at java.net.Socket.connect(Socket.java:470)
	at java.net.Socket.<init>(Socket.java:367)
	at java.net.Socket.<init>(Socket.java:209)
	at org.exoplatform.common.http.client.HTTPConnection.getSocket(HTTPConnection.java:3046)
	at org.exoplatform.common.http.client.HTTPConnection.sendRequest(HTTPConnection.java:2841)
	at org.exoplatform.common.http.client.HTTPConnection.handleRequest(HTTPConnection.java:2700)
	at org.exoplatform.common.http.client.HTTPConnection.setupRequest(HTTPConnection.java:2527)
	at org.exoplatform.common.http.client.HTTPConnection.Put(HTTPConnection.java:1063)
	at org.exoplatform.common.http.client.HTTPConnection.Put(HTTPConnection.java:1033)
	at org.exoplatform.common.http.client.HTTPConnection.Put(HTTPConnection.java:1012)
	at org.exoplatform.services.jcr.webdav.command.TestCopy.setUp(TestCopy.java:60)
	at junit.framework.TestCase.runBare(TestCase.java:125)
	at junit.framework.TestResult$1.protect(TestResult.java:106)
	at junit.framework.TestResult.runProtected(TestResult.java:124)
	at junit.framework.TestResult.run(TestResult.java:109)
	at junit.framework.TestCase.run(TestCase.java:118)
	at junit.framework.TestSuite.runTest(TestSuite.java:208)
	at junit.framework.TestSuite.run(TestSuite.java:203)
	at org.eclipse.jdt.internal.junit.runner.junit3.JUnit3TestReference.run(JUnit3TestReference.java:130)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:460)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:673)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:386)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:196)
29.10.2008 17:54:22 *ERROR* [main] HTTPConnection: Send request error (HTTPConnection.java, line 2933)
java.net.ConnectException: Connection refused
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.PlainSocketImpl.doConnect(PlainSocketImpl.java:333)
	at java.net.PlainSocketImpl.connectToAddress(PlainSocketImpl.java:195)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:182)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:366)
	at java.net.Socket.connect(Socket.java:520)
	at java.net.Socket.connect(Socket.java:470)
	at java.net.Socket.<init>(Socket.java:367)
	at java.net.Socket.<init>(Socket.java:209)
	at org.exoplatform.common.http.client.HTTPConnection.getSocket(HTTPConnection.java:3046)
	at org.exoplatform.common.http.client.HTTPConnection.sendRequest(HTTPConnection.java:2841)
	at org.exoplatform.common.http.client.HTTPConnection.handleRequest(HTTPConnection.java:2700)
	at org.exoplatform.common.http.client.HTTPConnection.setupRequest(HTTPConnection.java:2527)
	at org.exoplatform.common.http.client.HTTPConnection.Put(HTTPConnection.java:1063)
	at org.exoplatform.common.http.client.HTTPConnection.Put(HTTPConnection.java:1033)
	at org.exoplatform.common.http.client.HTTPConnection.Put(HTTPConnection.java:1012)
	at org.exoplatform.services.jcr.webdav.command.TestCopy.setUp(TestCopy.java:60)
	at junit.framework.TestCase.runBare(TestCase.java:125)
	at junit.framework.TestResult$1.protect(TestResult.java:106)
	at junit.framework.TestResult.runProtected(TestResult.java:124)
	at junit.framework.TestResult.run(TestResult.java:109)
	at junit.framework.TestCase.run(TestCase.java:118)
	at junit.framework.TestSuite.runTest(TestSuite.java:208)
	at junit.framework.TestSuite.run(TestSuite.java:203)
	at org.eclipse.jdt.internal.junit.runner.junit3.JUnit3TestReference.run(JUnit3TestReference.java:130)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:460)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:673)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:386)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:196)
29.10.2008 17:54:22 *ERROR* [main] HTTPConnection: Send request error (HTTPConnection.java, line 2933)
java.net.ConnectException: Connection refused
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.PlainSocketImpl.doConnect(PlainSocketImpl.java:333)
	at java.net.PlainSocketImpl.connectToAddress(PlainSocketImpl.java:195)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:182)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:366)
	at java.net.Socket.connect(Socket.java:520)
	at java.net.Socket.connect(Socket.java:470)
	at java.net.Socket.<init>(Socket.java:367)
	at java.net.Socket.<init>(Socket.java:209)
	at org.exoplatform.common.http.client.HTTPConnection.getSocket(HTTPConnection.java:3046)
	at org.exoplatform.common.http.client.HTTPConnection.sendRequest(HTTPConnection.java:2841)
	at org.exoplatform.common.http.client.HTTPConnection.handleRequest(HTTPConnection.java:2700)
	at org.exoplatform.common.http.client.HTTPConnection.setupRequest(HTTPConnection.java:2527)
	at org.exoplatform.common.http.client.HTTPConnection.Put(HTTPConnection.java:1063)
	at org.exoplatform.common.http.client.HTTPConnection.Put(HTTPConnection.java:1033)
	at org.exoplatform.common.http.client.HTTPConnection.Put(HTTPConnection.java:1012)
	at org.exoplatform.services.jcr.webdav.command.TestCopy.setUp(TestCopy.java:60)
	at junit.framework.TestCase.runBare(TestCase.java:125)
	at junit.framework.TestResult$1.protect(TestResult.java:106)
	at junit.framework.TestResult.runProtected(TestResult.java:124)
	at junit.framework.TestResult.run(TestResult.java:109)
	at junit.framework.TestCase.run(TestCase.java:118)
	at junit.framework.TestSuite.runTest(TestSuite.java:208)
	at junit.framework.TestSuite.run(TestSuite.java:203)
	at org.eclipse.jdt.internal.junit.runner.junit3.JUnit3TestReference.run(JUnit3TestReference.java:130)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:460)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:673)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:386)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:196)
29.10.2008 17:59:33 *ERROR* [main] HTTPConnection: Send request error (HTTPConnection.java, line 2933)
java.net.ConnectException: Connection refused
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.PlainSocketImpl.doConnect(PlainSocketImpl.java:333)
	at java.net.PlainSocketImpl.connectToAddress(PlainSocketImpl.java:195)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:182)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:366)
	at java.net.Socket.connect(Socket.java:520)
	at java.net.Socket.connect(Socket.java:470)
	at java.net.Socket.<init>(Socket.java:367)
	at java.net.Socket.<init>(Socket.java:209)
	at org.exoplatform.common.http.client.HTTPConnection.getSocket(HTTPConnection.java:3046)
	at org.exoplatform.common.http.client.HTTPConnection.sendRequest(HTTPConnection.java:2841)
	at org.exoplatform.common.http.client.HTTPConnection.handleRequest(HTTPConnection.java:2700)
	at org.exoplatform.common.http.client.HTTPConnection.setupRequest(HTTPConnection.java:2527)
	at org.exoplatform.common.http.client.HTTPConnection.Put(HTTPConnection.java:1063)
	at org.exoplatform.common.http.client.HTTPConnection.Put(HTTPConnection.java:1033)
	at org.exoplatform.common.http.client.HTTPConnection.Put(HTTPConnection.java:1012)
	at org.exoplatform.services.jcr.webdav.command.TestCopy.setUp(TestCopy.java:60)
	at junit.framework.TestCase.runBare(TestCase.java:125)
	at junit.framework.TestResult$1.protect(TestResult.java:106)
	at junit.framework.TestResult.runProtected(TestResult.java:124)
	at junit.framework.TestResult.run(TestResult.java:109)
	at junit.framework.TestCase.run(TestCase.java:118)
	at junit.framework.TestSuite.runTest(TestSuite.java:208)
	at junit.framework.TestSuite.run(TestSuite.java:203)
	at org.eclipse.jdt.internal.junit.runner.junit3.JUnit3TestReference.run(JUnit3TestReference.java:130)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:460)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:673)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:386)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:196)
29.10.2008 17:59:33 *ERROR* [main] HTTPConnection: Send request error (HTTPConnection.java, line 2933)
java.net.ConnectException: Connection refused
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.PlainSocketImpl.doConnect(PlainSocketImpl.java:333)
	at java.net.PlainSocketImpl.connectToAddress(PlainSocketImpl.java:195)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:182)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:366)
	at java.net.Socket.connect(Socket.java:520)
	at java.net.Socket.connect(Socket.java:470)
	at java.net.Socket.<init>(Socket.java:367)
	at java.net.Socket.<init>(Socket.java:209)
	at org.exoplatform.common.http.client.HTTPConnection.getSocket(HTTPConnection.java:3046)
	at org.exoplatform.common.http.client.HTTPConnection.sendRequest(HTTPConnection.java:2841)
	at org.exoplatform.common.http.client.HTTPConnection.handleRequest(HTTPConnection.java:2700)
	at org.exoplatform.common.http.client.HTTPConnection.setupRequest(HTTPConnection.java:2527)
	at org.exoplatform.common.http.client.HTTPConnection.Put(HTTPConnection.java:1063)
	at org.exoplatform.common.http.client.HTTPConnection.Put(HTTPConnection.java:1033)
	at org.exoplatform.common.http.client.HTTPConnection.Put(HTTPConnection.java:1012)
	at org.exoplatform.services.jcr.webdav.command.TestCopy.setUp(TestCopy.java:60)
	at junit.framework.TestCase.runBare(TestCase.java:125)
	at junit.framework.TestResult$1.protect(TestResult.java:106)
	at junit.framework.TestResult.runProtected(TestResult.java:124)
	at junit.framework.TestResult.run(TestResult.java:109)
	at junit.framework.TestCase.run(TestCase.java:118)
	at junit.framework.TestSuite.runTest(TestSuite.java:208)
	at junit.framework.TestSuite.run(TestSuite.java:203)
	at org.eclipse.jdt.internal.junit.runner.junit3.JUnit3TestReference.run(JUnit3TestReference.java:130)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:460)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:673)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:386)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:196)
29.10.2008 17:59:33 *ERROR* [main] HTTPConnection: Send request error (HTTPConnection.java, line 2933)
java.net.ConnectException: Connection refused
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.PlainSocketImpl.doConnect(PlainSocketImpl.java:333)
	at java.net.PlainSocketImpl.connectToAddress(PlainSocketImpl.java:195)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:182)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:366)
	at java.net.Socket.connect(Socket.java:520)
	at java.net.Socket.connect(Socket.java:470)
	at java.net.Socket.<init>(Socket.java:367)
	at java.net.Socket.<init>(Socket.java:209)
	at org.exoplatform.common.http.client.HTTPConnection.getSocket(HTTPConnection.java:3046)
	at org.exoplatform.common.http.client.HTTPConnection.sendRequest(HTTPConnection.java:2841)
	at org.exoplatform.common.http.client.HTTPConnection.handleRequest(HTTPConnection.java:2700)
	at org.exoplatform.common.http.client.HTTPConnection.setupRequest(HTTPConnection.java:2527)
	at org.exoplatform.common.http.client.HTTPConnection.Put(HTTPConnection.java:1063)
	at org.exoplatform.common.http.client.HTTPConnection.Put(HTTPConnection.java:1033)
	at org.exoplatform.common.http.client.HTTPConnection.Put(HTTPConnection.java:1012)
	at org.exoplatform.services.jcr.webdav.command.TestCopy.setUp(TestCopy.java:60)
	at junit.framework.TestCase.runBare(TestCase.java:125)
	at junit.framework.TestResult$1.protect(TestResult.java:106)
	at junit.framework.TestResult.runProtected(TestResult.java:124)
	at junit.framework.TestResult.run(TestResult.java:109)
	at junit.framework.TestCase.run(TestCase.java:118)
	at junit.framework.TestSuite.runTest(TestSuite.java:208)
	at junit.framework.TestSuite.run(TestSuite.java:203)
	at org.eclipse.jdt.internal.junit.runner.junit3.JUnit3TestReference.run(JUnit3TestReference.java:130)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:460)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:673)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:386)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:196)
29.10.2008 17:59:33 *ERROR* [main] HTTPConnection: Send request error (HTTPConnection.java, line 2933)
java.net.ConnectException: Connection refused
	at java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.net.PlainSocketImpl.doConnect(PlainSocketImpl.java:333)
	at java.net.PlainSocketImpl.connectToAddress(PlainSocketImpl.java:195)
	at java.net.PlainSocketImpl.connect(PlainSocketImpl.java:182)
	at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:366)
	at java.net.Socket.connect(Socket.java:520)
	at java.net.Socket.connect(Socket.java:470)
	at java.net.Socket.<init>(Socket.java:367)
	at java.net.Socket.<init>(Socket.java:209)
	at org.exoplatform.common.http.client.HTTPConnection.getSocket(HTTPConnection.java:3046)
	at org.exoplatform.common.http.client.HTTPConnection.sendRequest(HTTPConnection.java:2841)
	at org.exoplatform.common.http.client.HTTPConnection.handleRequest(HTTPConnection.java:2700)
	at org.exoplatform.common.http.client.HTTPConnection.setupRequest(HTTPConnection.java:2527)
	at org.exoplatform.common.http.client.HTTPConnection.Put(HTTPConnection.java:1063)
	at org.exoplatform.common.http.client.HTTPConnection.Put(HTTPConnection.java:1033)
	at org.exoplatform.common.http.client.HTTPConnection.Put(HTTPConnection.java:1012)
	at org.exoplatform.services.jcr.webdav.command.TestCopy.setUp(TestCopy.java:60)
	at junit.framework.TestCase.runBare(TestCase.java:125)
	at junit.framework.TestResult$1.protect(TestResult.java:106)
	at junit.framework.TestResult.runProtected(TestResult.java:124)
	at junit.framework.TestResult.run(TestResult.java:109)
	at junit.framework.TestCase.run(TestCase.java:118)
	at junit.framework.TestSuite.runTest(TestSuite.java:208)
	at junit.framework.TestSuite.run(TestSuite.java:203)
	at org.eclipse.jdt.internal.junit.runner.junit3.JUnit3TestReference.run(JUnit3TestReference.java:130)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:460)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:673)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:386)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:196)
